### ISSUE TITLE
TypeError __init__() got an unexpected keyword argument 'pooling_func'

### ISSUE THREAD

**Issue by vciulei** [NaegleLab/OpenEnsembles#12]
Hi, 

I have encountered the following error 
```
Exception has occurred: TypeError
__init__() got an unexpected keyword argument 'pooling_func'
```
while running this line:

`c.cluster(data_source, algorithm, out_name, K=k, Require_Unique=True,linkage=linkage, distance=dist,random_seed=seed)
`
This was generated by the Hierarchical Agglomerative Clustering algorithm, who I believe has been updated. I did not manage to find where in the source code I had to edit the call to the HAC, could you please point it out?

**Comment by knaegle** [comment]
Fixed in updates on commit 699034c018890779ced81ab2391990053ea96326 on py37 branch, to be merged to main.

**Comment by AWAIS59** [comment]
Hi,
I have same issue regarding when running this line:

c.cluster(data_source, algorithm, out_name, K=k, Require_Unique=True,linkage=linkage, distance=dist,random_seed=seed)

error raised is:
TypeError: __init__() got an unexpected keyword argument 'pooling_func'

Please guide me about its complete solution. how it would be fixed.
